charts/nginx-gateway-fabric/templates/deployment.yaml

```
@@ -15,14 +15,17 @@ spec:
     metadata:
       labels:
       {{- include "nginx-gateway.selectorLabels" . | nindent 8 }}
-      {{- if .Values.metrics.enable }}
       annotations:
+        {{- if .Values.nginxGateway.podAnnotations }}
+        {{- toYaml .Values.nginxGateway.podAnnotations | trim | nindent 8 }}
```

```suggestion
        {{- toYaml .Values.nginxGateway.podAnnotations | nindent 8 }}
```
I don't think we need the `trim`

---

You are right, trim is not needed so I removed it :)